src/vasp.jl

```diff
@@ -165,4 +165,59 @@ end
 
 readDOSCAR(filename::AbstractString) = open(readDOSCAR, filename)
 
-readDOSCAR() = open(readDOSCAR, "DOSCAR")
\ No newline at end of file
+readDOSCAR() = open(readDOSCAR, "DOSCAR")
+
+function readPROCAR(io::IO)
+    ln = readline(io)
+    
+    if cmp(split(ln)[2], "lm") == -1
+        error("Not a lm-decomposed PROCAR. Use a different PROCAR.")
+    end
+    
+    # Get number of kpoints, number of bands, number of ions
+    ln = readline(io)
+    nkpt = parse(Int,split(ln)[4])
+    nband = parse(Int,split(ln)[8])
+    nion = parse(Int,split(ln)[12])
+    
+    # Initializes containers... to be fixed
```

Comment:
FYI: if you want to initialize an array with all zeros, you can use
```julia
A = zeros(T, dims...)
```
which gives you an array of zeros of type `T` with the dimensions you specify. (1 dimension would be a vector, 2 would be a matrix, etc.) `zeros()` can be a safer choice, especially if not all array members are assigned in a loop and you need some sane default. (And there is also `ones()` and `fill()` for that purpose too)